modules/openthread/platform/radio.c

```diff
@@ -631,7 +631,7 @@ otError otPlatRadioSleep(otInstance *aInstance)
 	    sState == OT_RADIO_STATE_RECEIVE ||
 	    sState == OT_RADIO_STATE_TRANSMIT) {
 		error = OT_ERROR_NONE;
-		if (radio_api->stop(radio_dev)) {
+		if (radio_api->stop(radio_dev) == 0) {
```

Comment:
Please consider keeping consistency along all stop calls and either compare them all to 0 or remove this comparison.

Reply:
removed comparison to keep consistency with other api calls